src/js/index.js

```diff
@@ -488,6 +488,9 @@
     function calcBip32RootKeyFromSeed(phrase, passphrase) {
         seed = mnemonic.toSeed(phrase, passphrase);
         bip32RootKey = bitcoinjs.bitcoin.HDNode.fromSeedHex(seed, network);
+        if(isGRS())
+            bip32RootKey = groestlcoinjs.HDNode.fromSeedHex(seed, network);
+
     }
 
     function calcBip32RootKeyFromBase58(rootKeyBase58) {
```

Comment:
This method does get messy with all the exception handling.  We could make a separate method that handles only Groestlcoin and then call it if isGRS()==true